README.md

```diff
@@ -839,20 +839,47 @@ export default function Top() {
 
 ### Cloudflare Bindings
 
-If you want to use Cloudflare's Bindings in your development environment, create `wrangler.toml` and configure it properly.
-
-```toml
-name = "my-project-name"
-compatibility_date = "2024-04-01"
-compatibility_flags = [ "nodejs_compat" ]
-pages_build_output_dir = "./dist"
-
-# [vars]
-# MY_VARIABLE = "production_value"
+If you want to use Cloudflare's Bindings in your development environment, create `wrangler.jsonc` and configure it properly.
 
-# [[kv_namespaces]]
-# binding = "MY_KV_NAMESPACE"
-# id = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
+```jsonc
+// wrangler.jsonc
+{
+  "$schema": "node_modules/wrangler/config-schema.json",
+  "name": "my-project-name",
+  "main": "./dist/index.js",
+  "compatibility_date": "2024-04-01",
```

Comment:
It can be applied to other places. Updating `compatibility_date` is good. The one in the starter template is `2025-08-03`.

https://github.com/honojs/starter/blob/67a09f7121322bca10f305cb044804f916ea8957/templates/cloudflare-workers/wrangler.jsonc#L5